autoware.proj.repos

```diff
@@ -43,6 +43,10 @@ repositories:
     type: git
     url: https://github.com/tier4/ublox.git
     version: foxy-devel
+  vendor/geometry2:
```

Comment:
Could you also the a comment saying that this is just temporary for transforming covariances and should be removed once the backport is released?

Reply:
Never mind, other packages should be removed as well when there is new update so we can leave it as is.
Instead, I created new issue so that we don't forget to removed them after porting.

Reply:
https://github.com/tier4/AutowareArchitectureProposal/issues/104